### PR TITLE
[4.0] chore: Merge master branch and fix conflicts

### PR DESCRIPTION
**Pull request by dengfuping**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [x] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

- Ref: #19657.

### 💡 Background and solution

- 冲突比较多，有些合并策略不太确定，需要 CR @zombieJ.

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered CHANGELOG.en-US.md](https://github.com/ant-design/ant-design/blob/4.0-chore-merge-master/CHANGELOG.en-US.md)
[View rendered CHANGELOG.zh-CN.md](https://github.com/ant-design/ant-design/blob/4.0-chore-merge-master/CHANGELOG.zh-CN.md)
[View rendered components/config-provider/index.en-US.md](https://github.com/ant-design/ant-design/blob/4.0-chore-merge-master/components/config-provider/index.en-US.md)
[View rendered components/config-provider/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/4.0-chore-merge-master/components/config-provider/index.zh-CN.md)
[View rendered components/descriptions/demo/border.md](https://github.com/ant-design/ant-design/blob/4.0-chore-merge-master/components/descriptions/demo/border.md)
[View rendered components/descriptions/demo/responsive.md](https://github.com/ant-design/ant-design/blob/4.0-chore-merge-master/components/descriptions/demo/responsive.md)
[View rendered components/descriptions/demo/size.md](https://github.com/ant-design/ant-design/blob/4.0-chore-merge-master/components/descriptions/demo/size.md)
[View rendered components/descriptions/demo/vertical-border.md](https://github.com/ant-design/ant-design/blob/4.0-chore-merge-master/components/descriptions/demo/vertical-border.md)
[View rendered components/drawer/demo/user-profile.md](https://github.com/ant-design/ant-design/blob/4.0-chore-merge-master/components/drawer/demo/user-profile.md)
[View rendered components/drawer/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/4.0-chore-merge-master/components/drawer/index.zh-CN.md)
[View rendered components/form/index.en-US.md](https://github.com/ant-design/ant-design/blob/4.0-chore-merge-master/components/form/index.en-US.md)
[View rendered components/input/index.en-US.md](https://github.com/ant-design/ant-design/blob/4.0-chore-merge-master/components/input/index.en-US.md)
[View rendered components/input/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/4.0-chore-merge-master/components/input/index.zh-CN.md)
[View rendered docs/react/recommendation.en-US.md](https://github.com/ant-design/ant-design/blob/4.0-chore-merge-master/docs/react/recommendation.en-US.md)
[View rendered docs/react/recommendation.zh-CN.md](https://github.com/ant-design/ant-design/blob/4.0-chore-merge-master/docs/react/recommendation.zh-CN.md)

### CODE REVIEW

File: components/form/index.en-US.md [hunk 1]
```diff
@@ -16,28 +16,30 @@ High performance Form component with data scope management. Including data colle
...
...ms | boolean | false | | initialValues | Set value by Form initialization or reset | object | - | | labelAlign | Label text alignment | 'left' \| 'right' | 'right' | | labelCol | label layout, like `<Col>` component. Set `span` `offset` value like `{span: 3, offset: 12}` or `sm: {span: 3, offset: 12}` | [object](https://ant.design/components/grid/#Col) | | | layout | Form layout | 'horizontal'\|'vertical'\|'inline' | 'horizontal' | | name | Form name. Will be the prefix of Field `id` | string | - | | validateMessages | 验证提示模板，说明[见下](#validateMessages) | [ValidateMessages](https://github.com/react-component/field-form/blob/master/src/utils/messages.ts) | - | | wrapperCol | The layout for input controls, same as `labelCol` | [object](https://ant.design/components/grid/#Col) | | | onFinish | Trigger after submitting the form and verifying data successfully | Function(values) | - | | onFinishFailed | Trigger after submitting the form and verifying data failed | Function({ values, errorFields, outOfDate }) | - | | onFieldsChange | Trigger when field updated | Function(changedFields, allFields) | - | | onValuesChange | Trigger when value updated | Function(changedValues, allValues) | - |
```
Comment (yoyo837) [inline components/form/index.en-US.md]: <<<<<<<  冲突标记

File: components/form/index.en-US.md [hunk 2]
```diff
@@ -16,28 +16,30 @@ High performance Form component with data scope management. Including data colle
...
+| layout | Define form layout | 'horizontal'\|'vertical'\|'inline' | 'horizontal' |  |
+| onSubmit | Defines a function will be called if form data validation is successful. | Function(e:Event) |  |  |
+| wrapperCol | (Added in 3.14.0. Previous version can only set on FormItem.) The layout for input controls, same as `labelCol` | [object](https://ant.design/components/grid/#Col) |  | 3.14.0 |
+| colon | change default props colon value of Form.Item (only effective when prop layout is horizontal) | boolean | true | 3.15.0 |
+
+### Form.create(options)
+
+How to use：
+
+> > > > > > > master
```
Comment (afc163) [inline components/form/index.en-US.md]: 冲突

**Comment by zombieJ** [comment]
我重新搞一下

**Comment by yoyo837** [comment]
#19709

**Comment by zombieJ** [comment]
Updated.

**Pull request merged**